CHANGELOG.md

```diff
@@ -0,0 +1,134 @@
+# TiSpark Changelog
+All notable changes to this project will be documented in this file.
+
+## [TiSpark 2.2.0]
+### New Features
+* Natively support writing data to TiKV using Spark Data Source API
+* Support select from partition table [#916](https://github.com/pingcap/tispark/pull/916)
+* Release one tispark jar instead of two [#933](https://github.com/pingcap/tispark/pull/933)
```

Comment:
Should we add something like:

Support Spark 2.3/2.4

Reply:
fixed